src/pages/404.js

```diff
@@ -7,28 +7,75 @@ import styled from 'styled-components'
 import remcalc from 'remcalc'
 import ReactMarkdown from 'react-markdown'
 
+import BlueBackground from '../components/Common/BlueBackground'
 import { SectionTitle, BodyPrimary } from '../components/Typography'
-import Button from '../components/Common/Button'
 import Layout from '../components/layout'
 
-const HomePageLink = styled(Button)`
-  width: ${remcalc(145)};
-  display: flex;
-  justify-content: center;
+import { LogoStyleContext } from '../context/PageContext'
+
+import illustration from '../images/404-illustration.svg'
+
+// Layout
+
+const StyledBlueBackground = styled(BlueBackground)`
+  margin-top: -${remcalc(36)};
 `
 
-const StyledBodyPrimary = styled(BodyPrimary)`
-  padding-bottom: ${({ theme }) => theme.space[6]};
+const StyledRow = styled(Row)`
+  align-items: center;
 `
 
-const StyledCol = styled(Col)`
-  margin: ${({ theme }) => theme.space[5]} 0;
+const TextCol = styled(Col).attrs({
+  width: [1, 1, 1, 1, 0.5, 0.5]
+})`
+  padding-top: ${({ theme }) => theme.space[4]};
+  ${breakpoint('smallTablet')`
+    padding-bottom: ${({ theme }) => theme.space[4]};
+  `}
+  ${breakpoint('tablet')`
+    padding-top: ${({ theme }) => theme.space[7]};
+    padding-bottom: ${({ theme }) => theme.space[8]};
+  `}
+`
 
+const IllustrationCol = styled(Col).attrs({
+  width: [1, 1, 1, 1, 0.5, 0.5]
+})`
+  ${breakpoint('smallTablet')`
+    padding-top: ${({ theme }) => theme.space[4]};
+    padding-bottom: ${({ theme }) => theme.space[4]};
+  `}
   ${breakpoint('tablet')`
-    margin: ${({ theme }) => theme.space[8]} 0
+    padding-top: unset;
+    padding-bottom: unset;
   `}
 `
 
+// Paragraphs / Texts
+
+const TitleHeadline = props => (
+  <SectionTitle {...props} as="h1" reverse="true" />
+)
+
+const CopyText = props => <BodyPrimary {...props} reverse="true" />
+
+const LinksTitle = styled(BodyPrimary).attrs({
+  reverse: 'true',
+  bold: 'true'
+})`
+  padding-top: ${({ theme }) => theme.space[3]};
+`
+
+const LinkParagraph = styled('p')`
+  padding-top: ${({ theme }) => theme.space[2]};
+`
+
+const NotFoundPageLink = styled(Link)`
+  text-decoration: underline;
+`
```

Comment:
Same as above, I believe we have `UnstyledLink` available for this instead of using the gatsby exporting component

Reply:
Can't find `UnstyledLink`, and `StyledLink` has a lot of styling / `::after` I'd need to get rid of (used that at first)

Reply:
Ok cool, lets see what design says, should be fine